server/services/participants.js

```diff
@@ -125,6 +125,52 @@ const updateParticipant = async (participantInfo) => {
   return participant;
 };
 
+const validateConfirmationId = (id) =>
+  dbClient.db[collections.CONFIRM_INTEREST].findOne({ otp: id });
```

Comment:
Is this necessary? If the OTP is invalid then the relatedParticipants query would be null. Calling it a nitpick as removing this would require catching it elsewhere.

Reply:
This is the bit that we call to see if we should show the invalid message, are you saying we could just call the other function below to get the same effect?

Reply:
It only confirms that the uuid is in the table, when I would have just queried the table and checked afterwards to see if it hit anything.